--- a/audio/data.py
+++ b/audio/data.py
@@ -1,4 +1,4 @@
-from dataclasses import dataclass, asdict
+from dataclasses import dataclass, asdict, field
 from IPython.display import Audio
 import torch.nn.functional as F
 from pathlib import Path
@@ -48,7 +48,7 @@
     _sr = None
     _nchannels = None
 
-    sg_cfg: SpectrogramConfig = SpectrogramConfig()
+    sg_cfg: SpectrogramConfig = field(default_factory=SpectrogramConfig)
 
     def __setattr__(self, name, value):
         '''Override to warn user if they are mixing seconds and ms'''
